frontend/temp/index.html

```diff
@@ -11,13 +11,14 @@
     <!-- inject:css -->
```

Comment:
The files from /temp folder shouldn't be committed / modified as they are generated when you run gulp. I'm sorry for the trouble, I shouldn't have committed it here, and I should've added it to .gitignore. Feel free to do it yourself (and u can also push changes of removing the temp folder from this repo).